app/src/main/java/com/duckduckgo/app/browser/favorites/NewTabLegacyPageViewModel.kt

```diff
@@ -61,7 +63,7 @@ import kotlinx.coroutines.withContext
 @ContributesViewModel(ViewScope::class)
 class NewTabLegacyPageViewModel @Inject constructor(
     private val dispatchers: DispatcherProvider,
-    private val remoteMessagingModel: Lazy<RemoteMessagingModel>,
+    private val remoteMessagingModel: RemoteMessageModel,
```

Comment:
Why not lazy anymore?

Reply:
It doesn’t need to be lazy, that was an unnecessary step.